Update Twitter Documentation to Mention API pricing (#208)
quackduck/devzat: Admin's Manual.md

    compact_mode: true # optional: disables avatars so messages take up less vertical space
```

#### Using the X integration

> Devzat supports sending updates about who is online on the X social media platform. However, because of the switch from free to paid API keys, This feature no longer is no longer updated.

Reviewer: Why do you think it won't be updated? It might be.

Author: changed that

> ![API Key Pricing](https://global.discourse-cdn.com/twitter/original/3X/b/4/b42b6b7e4db2db3f51db70b439e9f111105db04d.png)

Reviewer: We can let people find pricing themselves. Simply mentioning that the api is now paid and then linking to the new app page is enough

```

#### Using the Twitter integration

> Devzat supports sending updates about who is online on Twitter. Start by creating a new app through a [Twitter developer account](https://developer.twitter.com/en/apply/user) (note: Twitter's API is now paid).

Reviewer: ```suggestion
Devzat supports posting updates about who is online to Twitter. Start by creating a new app through a [Twitter developer account](https://developer.twitter.com/en/apply/user) (note: Twitter's API is now paid).
```
